locales/puppet.pot

```diff
@@ -0,0 +1,128 @@
+# SOME DESCRIPTIVE TITLE.
```

Comment:
I think these might be coming from the underlying system tools. gettext-setup passes a number of arguments to `rxgettext` [here](https://github.com/puppetlabs/gettext-setup-gem/blob/master/lib/tasks/gettext.rake#L36-L41), which seems to cover most of the [available options](https://gist.github.com/Magisus/2a26519e72a4c84df9be6a5e9aaa51d5). There are also [these docs](https://www.gnu.org/software/gettext/manual/html_node/Header-Entry.html#Header-Entry) which makes it sound like it has to be done manually...